### PR TITLE
fix(timezone): fixes wrong timezone in replies

### LINKED ISSUE #34: Different time zone is being used in new twts and in replies
Create a new twt, and then reply to it.

```
2024-12-13T21:39:32Z	A test!
2024-12-13T16:51:20-05:00	(#vborwuq) a reply to that test
```

We could see that the new twt uses Zulu timezone, and the reply uses `-05:00` which is a weird choice since the sandbox is set for [Europe/Copenhagen UTC+1](https://www.zeitverschiebung.net/en/timezone/europe--copenhagen)

Edit: I found the server it set to [America/New_York UTC-5](https://www.zeitverschiebung.net/en/timezone/america--new_york)

Live example: https://twtxt-sandbox.eapl.me/twtxt.txt - Lines 37-38

### PR DESCRIPTION
**Pull request by eapl-gemugami**
Fixes #34
- Move loading timezone from config file to `load_timezone.php` script
- Apply code style suggestions from IDE

### CODE REVIEW

File: private/config_template.ini [hunk 1]
```diff
@@ -46,9 +51,9 @@ public_webmentions = "https://example.com/timeline/mentions.txt"
 totp_digits = 10
 totp_secret = "1234567890"
 
+; Simple password for unnamed user
+password = "123"
```
Comment (sorenpeter) [inline private/config_template.ini:55]: It does not seem very secure to set a default password. If no password it set it should trow an error or we should make a setup/settings page that the user will be redirect to on first visit after installing. I already stared making a mockup based on config.ini

Comment (eapl-gemugami) [reply private/config_template.ini:55]: I agree that a setup script, or a verification after running any page would help to stop known errors.

I started a short verification of the config.ini here (only for sessions)
https://github.com/sorenpeter/timeline/blob/8b133883d4a43ccccfa98790cfa33adf50c58d8d/libs/persistent_session.php#L4-L6
It could help you to check that some keys have values (specially on code updates), and also as in the password we'd have to check for some specific values (for example, password should be at least N characters).

Also I've tested using a default empty password `password = ""`, and here I added a validation so it won't work.
https://github.com/sorenpeter/timeline/blob/8b133883d4a43ccccfa98790cfa33adf50c58d8d/libs/session.php#L34
But again, having those validations in a general "config checker" would be better.

**Comment by eapl-gemugami** [comment]
@sorenpeter any comments for this PR?

**Review by sorenpeter — APPROVED** [review]
